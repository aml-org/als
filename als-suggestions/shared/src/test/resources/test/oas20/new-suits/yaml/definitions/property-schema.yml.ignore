swagger: '2.0'
definitions:
  name:
    type: object
    properties:
      p1:
        *