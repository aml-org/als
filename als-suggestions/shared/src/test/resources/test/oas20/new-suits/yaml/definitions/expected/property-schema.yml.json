[
  {
    "text": "required:\n          ",
    "description": "",
    "displayText": "required",
    "prefix": "",
    "category": "schemas",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "enum: ",
    "description": "",
    "displayText": "enum",
    "prefix": "",
    "category": "parameters",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "type: ",
    "description": "Required. The type of the parameter. Since the parameter is not located at the request body, it is limited to simple types (that is, not an object). The value MUST be one of \"string\", \"number\", \"integer\", \"boolean\", \"array\" or \"file\". If type is \"file\", the consumes MUST be either \"multipart/form-data\" or \" application/x-www-form-urlencoded\" and the parameter MUST be in \"formData\".",
    "displayText": "type",
    "prefix": "",
    "category": "schemas",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "xml:\n          ",
    "description": "",
    "displayText": "xml",
    "prefix": "",
    "category": "schemas",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "default: ",
    "description": "",
    "displayText": "default",
    "prefix": "",
    "category": "parameters",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "pattern: ",
    "description": "",
    "displayText": "pattern",
    "prefix": "",
    "category": "parameters",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "description: ",
    "description": "",
    "displayText": "description",
    "prefix": "",
    "category": "docs",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "maxLength: ",
    "description": "",
    "displayText": "maxLength",
    "prefix": "",
    "category": "parameters",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "title: ",
    "description": "",
    "displayText": "title",
    "prefix": "",
    "category": "docs",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "minLength: ",
    "description": "",
    "displayText": "minLength",
    "prefix": "",
    "category": "parameters",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "\"$ref\": ",
    "description": "",
    "displayText": "$ref",
    "prefix": "",
    "category": "schemas",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "format: ",
    "description": "",
    "displayText": "format",
    "prefix": "",
    "category": "parameters",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  },
  {
    "text": "example: ",
    "description": "",
    "displayText": "example",
    "prefix": "",
    "category": "docs",
    "range": [
      {
        "start": {
          "line": 6,
          "character": 8
        },
        "end": {
          "line": 6,
          "character": 8
        }
      }
    ]
  }
]